apps/nextjs/src/hooks/useMobileLessonPullOutControl.ts

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react";
+
+import { Message } from "ai";
+
+import { AilaStreamingStatus } from "@/components/AppComponents/Chat/Chat/hooks/useAilaStreamingStatus";
+
+export const useMobileLessonPullOutControl = ({
```

Comment:
What actually is the mobile lesson pullout? can't see it anywhere otherthan "close" or "user has overridden". Is it a component?

Reply:
It's the right-hand side that becomes a pull out for mobile